api/src/main/java/org/openmrs/validator/AllergyValidator.java

```diff
@@ -25,57 +25,65 @@
 @Component("allergyValidator")
 @Handler(supports = { Allergy.class }, order = 50)
 public class AllergyValidator implements Validator {
-	
+
```

Comment:
Thanks @Muta-Jonathan for attempting this ,
you seem to have too many formating changes , can you clean them up ??